README.md

```diff
@@ -43,6 +36,17 @@ CURL used to include a list of accepted CAs, but no longer bundles ANY CA certs.
 ;curl.cainfo =
 ```
 
+## How to Test
+
+Unit tests in this SDK can be run using PHPUnit. The test cases are located in the test/Controllers/ dir.
+
+1. Make sure you've installed the dependencies using composer including the `require-dev` dependencies.
+1. Run `vendor/bin/phpunit` from command line to execute the test suite.
+   See https://phpunit.de/manual/current/en/textui.html for info on test output format as well as more command-line options.
+1. Optionally, to check the code standards you may run `vendor/bin/phpcs --standard=PSR1 test`. (No output means no problems.)
```

Comment:
Moved this block to after **How to Build** and added `phpcs` instructions to lint with PSR-1.